scripts/copyResearchDocs.ts

```diff
@@ -59,9 +73,16 @@ import { adapters, type Tracker } from 'trackhar';
     );
     await cp('trackhar_tmp/research-docs', 'content/en/research', { recursive: true });
 
+    console.info('Generate new example data…');
+    await execa('yarn', [], { cwd: 'trackhar_tmp' });
+    // If the server is too slow, we should timeout after 5 minutes
+    await execa('yarn', ['tsx', 'scripts/generate-example-data.ts'], { cwd: 'trackhar_tmp', timeout: 300000 });
```

Comment:
I feel like we need to double this. I've already had a run that took a little longer than 5 mins.